evm/vm/base.py

```diff
@@ -344,13 +343,7 @@ def create_unsigned_transaction(self, *args, **kwargs):
     #
     @classmethod
     def get_block_class(cls):
-        """
-        Return the class that this VM uses for blocks.
```

Comment:
Should I put the comment back in? Not sure if this is meant to have a comment now that it only delegates to `get_block_class()` on the state?

Reply:
I think the comment should be changed to indicate that it's a passthrough to to the underlying API.

Reply:
:+1: I updated it